pex/bin/pex.py

```diff
@@ -958,15 +958,21 @@ def _compatible_with_current_platform(interpreter, platforms):
 def main(args=None):
     args = args[:] if args else sys.argv[1:]
     args = [transform_legacy_arg(arg) for arg in args]
-    parser = configure_clp()
+
+    parser = catch(configure_clp)
```

Comment:
Testing out the hiddenness of pep24 from the production CLI `--pip-version` support revealed these two spots where raise exceptions led to noise backtraces instead of clean bad value messages.